test/toPairs.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+var throws = require('assert').throws;
+
+var eq = require('./utils').eq;
+var errorEq = require('./utils').errorEq;
+var S = require('..');
+
+
+describe('toPairs', function() {
+
+  var comparePairsAsc = function(a, b) {
+    if (a[0] > b[0]) return 1;
+    if (a[0] < b[0]) return -1;
+    return 0;
```

Comment:
I'd prefer `a[0] < b[0] ? -1 : a[0] > b[0] ? 1 : 0` but I'm aware that not everyone is as fond of ternary expressions as I am. :)